libfreerdp/core/listener.c

```diff
@@ -347,8 +352,10 @@ static BOOL freerdp_listener_check_fds(freerdp_listener* instance)
 
 	for (i = 0; i < listener->num_sockfds; i++)
 	{
+		WSAResetEvent(listener->events[i]);
```

Comment:
Oops. You've already re-defined WSAResetEvent. It should be correct now.
Although it still looks a little weird and someone read this line may think it as a bug: why we call a function which does nothing. :)